ezbake/config/logback.xml

```diff
@@ -18,6 +18,7 @@
 
     <root level="info">
         <!--<appender-ref ref="STDOUT"/>-->
+        <appender-ref ref="${logappender:-DUMMY}"/>
```

Comment:
Hmm....I'm not sure that we need to provide a default value here. I seem to be able to run everything as expected without the `:-DUMMY` part.